egs/thchs30/s5/path.sh

```diff
@@ -0,0 +1,18 @@
+#Change the following settings to match your computing env,
+#so that appropriate (CPU/GPU) Kaldi commands are visible
+
+export KALDI_ROOT=/work3/zxw/kaldi-2016-1-21
+
+if [ -z `which nvidia-smi 2>/dev/null` ];then
+#if nvidia_smi; then
+    src=src_nocuda
+    echo "using no-cuda kaldi src: $KALDI_ROOT src"
+else
+    src=src_cuda
+    echo "using cuda kaldi src: $KALDI_ROOT src"
+fi
+
```

Comment:
The src_nocuda/src_cuda will be perhaps confusing for other people than you (as the default directory is just "src", so no matter what path will be chosen (depending on existence nvidia-smi), it will always generate an incorrect $PATH
Also, ad the "which"  command -- you might want possibly consider using "command -v nvidia-smi >/dev/null 2>&1", which is POSIX defined -- "which" is sometimes hijacked by packagers (on ubuntu, for example) and that might lead to confusion. But I leave this on you and Dan.